client/notification_test.go

```diff
@@ -85,6 +87,25 @@ var _ = Describe("Notification Client", func() {
 				Expect(*createdNotification).To(Equal(mockNotification))
 			})
 		})
+
+		Describe("Failure - Multiple Organizations", func() {
```

Comment:
This is very annoying... but I can't test in organization_test.
organizationId() function is private and therefore can be tested (test is using a different package).
(In general tests shouldn't be using different package names, but at this stage it will be difficult to modify).

To work around that I picked a random API call that calls organizationId (E.g. NotificationCreate).
I mocked "/organizations" which is called by organizationId which is called by NotificationCreate.

This now tests the use-case... just a little ugly.

Reply:
can you please add that in a comment? So the next person who works on this will understand what this is doing here

Reply:
@avnerenv0 - done:
https://github.com/env0/terraform-provider-env0/pull/449/commits/43991c6c76044c923a3098826408db478c69294f